draft-ietf-acme-acme.md

```diff
@@ -521,6 +521,53 @@ set to a URI other than those defined above.  Servers MUST NOT use the ACME URN
 namespace for errors other than the standard types.  Clients SHOULD display the
 "detail" field of all errors.
 
+### Sub-problems
+
+Sometimes a CA may need to return multiple errors to a single
```

Comment:
nitpicky nit: "return multiple errors to a request" reads better in my mind as "return multiple errors in response to a request"